ocean_provider/routes.py

```diff
@@ -86,7 +86,6 @@ def simple_flow_consume():
         'transferTxId'
     ]
     data = get_request_data(request)
-
```

Comment:
Please add this line back. Files not affected by commit should not be changed, and the empty line serves for code readability and delimitation.